controller/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const { deleteCommentFromDB, checkCommentExists } = require("../model/comments.models")
+
+function deleteCommentById(req,res,next){
+    const {comment_id}=req.params
+// deleteCommentFromDB(comment_id).then((result)=>{
+//     res.status(204).send({})
+// }).catch(next)
+    const promises=[deleteCommentFromDB(comment_id),checkCommentExists(comment_id)]
```

Comment:
It's not a good idea to use a Promise.all for this. You're deleting the comment and checking if it exists at the same time and either of these could resolve first. For this endpoint, please check the comment exists and then delete it.